README.md

```diff
@@ -61,6 +65,10 @@ This notebook is intended to demonstrate the capabilities of the "ariaPlot.py" p
 This notebook provides an overview of the functionality of the "ariaTSsetup.py" program. Specifically, we give examples on how to extract data and meta-data layers from ARIA Geocoded UNWrapped interferogram (GUNW) products over a user defined area of interest and prepare the data into a stack for time-series ingestion.
 + [ariaTSsetup](https://nbviewer.jupyter.org/github/aria-tools/ARIA-tools-docs/blob/master/JupyterDocs/ariaTSsetup/ariaTSsetup_tutorial.ipynb)
 
+### Assessing phase triplet misclosure - ariaMisclosure.py program tutorial
+This notebook demonstrates the "ariaMisclosure.py" tool for determining phase triplet misclosure. It begins with a brief overview of the concept of phase misclosure. It then demonstrates dynamic uses of the tool for formulating phase triplets and visualizing the results.
++ [ariaMisclosure](https://nbviewer.jupyter.org/github/aria-tools/ARIA-tools-docs/blob/master/JupyterDocs/ariaMisclosure/ariaMisclosure_tutorial.ipynb)
+
```

Comment:
@rzinke  is the notebook purely for information/statistics only? If possible could you add one more sentences detailing how one could be using this information for time-series analysis?